ios/browser/api/ntp_background_images/ntp_sponsored_image.mm

```diff
@@ -147,7 +147,7 @@ - (instancetype)initWithSponsoredBackground:
 @end
 
 @interface NTPSponsoredImageLogo ()
-@property(nonatomic, copy) NSURL* imagePath;
+@property(nonatomic, copy, nullable) NSURL* imagePath;
```

Comment:
@aseren I suggest we keep the changes to a minimum on iOS. I agree that nullable is necessary, but the other changes don’t seem essential to me unless I’m overlooking something.

Reply:
Fixed